tools/build-wheels.sh

```diff
@@ -26,28 +31,35 @@ done
 echo
 echo
 echo "Bundle external shared libraries into the wheels"
-for whl in /io/dist/${package_name}*${arch}.whl; do
+for whl in /io/dist/${package_name}-*-linux_${arch}.whl; do
     echo "Repairing $whl..."
     auditwheel repair "$whl" -w /io/dist/
 done
 
+echo
+echo
 echo "Cleanup OS specific wheels"
 rm -fv /io/dist/*-linux_*.whl
 
 echo
 echo
+echo "Cleanup non-$package_name wheels"
+find /io/dist -maxdepth 1 -type f ! -name "$package_name"'-*-manylinux1_'"$arch"'.whl' -delete
+
+echo
+echo
 echo "Install packages and test"
 echo "dist directory:"
 ls /io/dist
 
 for PYTHON in ${PYTHON_VERSIONS}; do
+    # clear python cache
+    find /io -type d -name __pycache__ -delete
```

Comment:
Does `-delete` remove directories?
I thought it works for files only

Reply:
Oops, you're right. I'll fix this